### PR TITLE
accept 'magic 8 ball' as input

### PR DESCRIPTION
**Pull request by Delanoche**
this way is better

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ app.route('/')
   .post(function(req, res) {
     if(req.body.name.toLowerCase().indexOf('magic eight ball') < 0 && req.body.text.toLowerCase().indexOf('magic eight ball') > -1) {
       setTimeout(sayBot(res), 4000);
+    }else if(req.body.name.toLowerCase().indexOf('magic 8 ball') < 0 && req.body.text.toLowerCase().indexOf('magic 8 ball') > -1) {
```
Comment (dipernaa) [inline server.js]: This else if statement does the same thing. Why not just extract the check that it is doing into a function and feed that function the different names you want to accept? That way you avoid duplication and smaller line lengths.

**Comment by Delanoche** [comment]
# SIGH

**Comment by dipernaa** [comment]
I'm just trying to make the best magic eight ball for the people

**Comment by Delanoche** [comment]
Now that I read this, why do you do `< 0 && > -1` as opposed to `== 0`?

**Comment by dipernaa** [comment]
I prefer to use less thans or greater thans instead of equals. That way I'm covering more cases.

**Comment by Delanoche** [comment]
...

**Comment by dipernaa** [comment]
I need to explain this in person. I look at it as that I only care that it is not a certain value. It can go be whatever it wants to be, just not greater than or less than the value I am expecting. Does that make sense?

**Comment by Delanoche** [comment]
I mean you could be more wrong, but it's still wonky.

**Comment by dipernaa** [comment]
You're wonky

**Comment by Delanoche** [comment]
Does this tickle you more in the right places?

**Comment by dipernaa** [comment]
# Mmmm... yes.

**Pull request merged**